lib/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components",
-  "version": "7.1.2",
+  "version": "7.1.3",
```

Comment:
This would be a minor version

Reply:
> This would be a minor version

Minor version bumped instead in https://github.com/sonatype/sonatype-react-shared-components/pull/418/commits/ad865014ca44474766838735c2cd9ac8beead1bf

Reply:
> Minor version bumped instead in ad86501

Should be 7.2.0.  Please use `yarn version --no-git-tag-version --minor`or similar  in the `lib` directory for version bumps

Reply:
> Should be 7.2.0. Please use `yarn version --no-git-tag-version --minor`or similar in the `lib` directory for version bumps

Fixed in https://github.com/sonatype/sonatype-react-shared-components/pull/418/commits/7e78c6d3b9bc49b11756ec499cf15244ff796b0f